### PR TITLE
[WIP] [DO NOT REVIEW] Experimental etcd setup

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1284,6 +1284,11 @@ function prepare-etcd-manifest {
 
   local -r temp_file="/tmp/$5"
   cp "${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty/etcd.manifest" "${temp_file}"
+  if [[ "suffix" == "" ]]; then
+    sed -i -e "s@{{ *data_directory *}}@/var/etcd/data@g" "${temp_file}"
+  else
+    sed -i -e "s@{{ *data_directory *}}@/tmp/data@g" "${temp_file}"
```
Comment (wojtek-t) [inline cluster/gce/gci/configure-helper.sh]: Actually, this won't be tmpfs - it will be local disk

Comment (wojtek-t) [reply cluster/gce/gci/configure-helper.sh]: Actually this PR doesn't make any sense...

Comment (shyamjvs) [reply cluster/gce/gci/configure-helper.sh]: Increasing disk to 500GB part probably helps. But even I'm not sure if/how `/tmp` works.

**Pull request closed without merging**